ci/nvim-lspconfig/default.nix

```
@@ -19,7 +19,10 @@ let
     ];
   };
 
-  nvim = wrapNeovimUnstable neovim-unwrapped nvimConfig;
+  nvim = (wrapNeovimUnstable neovim-unwrapped nvimConfig).overrideAttrs (_: {
+    dontFixup = true;
+  });
```

`overrideAttrs` can receive _either_ a function _or_ a set. No need to pass a function and ignore its input.

```suggestion
  nvim = (wrapNeovimUnstable neovim-unwrapped nvimConfig).overrideAttrs {
    dontFixup = true;
  };
```